test/distribution/test_distribution_dirichlet_static.py

```diff
@@ -120,3 +121,113 @@ def test_entropy(self):
                 rtol=RTOL.get(str(self.concentration.dtype)),
                 atol=ATOL.get(str(self.concentration.dtype)),
             )
+
+
+@place(DEVICES)
+@parameterize_cls(
+    (TEST_CASE_NAME, 'concentration'),
+    [('test-one-dim', np.random.rand(89) + 5.0)],
+)
+class TestDirichletPir(unittest.TestCase):
+    def setUp(self):
+        with paddle.pir_utils.IrGuard():
+            self.program = paddle.static.Program()
+            self.executor = paddle.static.Executor()
+            with paddle.static.program_guard(self.program):
+                conc = paddle.static.data(
+                    'conc', self.concentration.shape, self.concentration.dtype
+                )
+                self._paddle_diric = paddle.distribution.Dirichlet(conc)
+                self.feeds = {'conc': self.concentration}
+
+    @test_with_pir_api
+    def test_mean(self):
+        with paddle.static.program_guard(self.program):
+            [out] = self.executor.run(
+                self.program,
+                feed=self.feeds,
+                fetch_list=[self._paddle_diric.mean],
+            )
+            np.testing.assert_allclose(
+                out,
+                scipy.stats.dirichlet.mean(self.concentration),
+                rtol=RTOL.get(str(self.concentration.dtype)),
+                atol=ATOL.get(str(self.concentration.dtype)),
+            )
```

Comment:
```suggestion
    def test_mean(self):
        with paddle.pir_utils.IrGuard():
            with paddle.static.program_guard(self.program):
                [out] = self.executor.run(
                    self.program,
                    feed=self.feeds,
                    fetch_list=[self._paddle_diric.mean],
                )
                np.testing.assert_allclose(
                    out,
                    scipy.stats.dirichlet.mean(self.concentration),
                    rtol=RTOL.get(str(self.concentration.dtype)),
                    atol=ATOL.get(str(self.concentration.dtype)),
                )
```
此处不宜使用 `@test_with_pir_api`，被 `@test_with_pir_api`  修饰的单测会在旧 ir 模式和 pir 模式下分别运行一次。因为该单测为 pir 单测，所以我们只需要在 Pir 模式下运行就好了。使用 `paddle.pir_utils.IrGuard()` 作为上下文管理器可以切换到 pir 模式进行组网